duckduckgo/Android
# Update widget to match new header

Task/Issue URL: https://app.asana.com/0/488551667048375/619730419617923

**Description**:
Update the android search widget colors and add increase rounding to better match new header

**Steps to test this PR**:
1. Install the app
1. Add the widget to the launcher
1. Ensure both the preview and widget have the new color scheme

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
